.travis.yml

```diff
@@ -1,7 +1,22 @@
-sudo: false
+dist: trusty
+sudo: required
 language: node_js
 node_js:
 - '10'
+addons:
+  firefox: latest
```

Comment:
are we testing on firefox?

Reply:
no we only test on chrome. I think I should remove that

Reply:
actually...I dont think we can remove it. the chrome stuff is part of it i think?